operators/olm_v1/arch/olmv1-components.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="olmv1-components"]
+= {olmv1} components overview (Technology Preview)
+:context: olmv1-components
+
+toc::[]
+
+:FeatureName: {olmv1}
+include::snippets/technology-preview.adoc[]
+
+{olmv1-first} comprises the following component projects:
+
+Operator Controller:: xref:../../../operators/olm_v1/arch/olmv1-operator-controller.adoc#olmv1-operator-controller[Operator Controller] is the central component of {olmv1} that extends Kubernetes with an API through which users can install and manage the lifecycle of Operators and extensions. It consumes information from each of the following components.
+
+RukPak:: xref:../../../operators/olm_v1/arch/olmv1-rukpak.adoc#olmv1-rukpak[RukPak] is a pluggable solution for packaging and distributing cloud-native content. It supports advanced strategies for installation, updates, and policy.
++
+RukPak provides a content ecosystem for installing a variety of artifacts, such as Git repositories, Helm charts, and OLM bundles, on a Kubernetes cluster. The component can then manage, scale, and upgrade these artifacts in a safe way to enable powerful cluster extensions.
++
+At its core, RukPak is a small set of APIs, packaged as custom resource definitions (CRDs), and controllers that watch for those APIs. The APIs express content to install on a cluster and how to create a running deployment of the content.
```

Comment:
```suggestion
At its core, RukPak is a small set of APIs and controllers. The APIs are packaged as custom resource definitions (CRDs) that express what content to install on a cluster and how to create a running deployment of the content. The controllers watch for the APIs.
```
Maybe that changes the meaning too much, but I was getting a little tangled up in the original.

Reply:
> I was getting a little tangled up

Me too. Your change keeps my intended meaning/understanding, FWIW, will check during SME review to make sure still accurate.